crates/router/src/connector/square.rs

```diff
@@ -149,6 +151,15 @@ impl ConnectorIntegration<api::Authorize, types::PaymentsAuthorizeData, types::P
         _req: &types::PaymentsAuthorizeRouterData,
         _connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
+        // should uncomment below if block once implemented
+        // if req.request.capture_method == Some(enums::CaptureMethod::ManualMultiple) {
+        //     return Err(errors::ConnectorError::NotImplemented(format!(
+        //         "{}{}",
+        //         consts::MANUAL_MULTIPLE_NOT_IMPLEMENTED_ERROR_MESSAGE,
+        //         self.id()
+        //     ))
+        //     .into());
+        // }
```

Comment:
you can have the validation now itself, since the validation is moved to the different trait